database/revert/V20170114_2116__createAttendeeRSVP.revert.sql

```diff
@@ -1 +1,5 @@
-DROP TABLE `attendee_rsvps`;
\ No newline at end of file
+DROP TABLE `attendee_rsvps`;
+DELETE FROM `mailing_lists` WHERE `name`='attendees';
```

Comment:
No, we're not reverting the migration, but since each migration has a revert in case we need to revert something the revert should properly reverse the changes the migration makes.